R/mod_user_attributes.R

```diff
@@ -86,8 +86,9 @@ mod_user_attributes_server <- function(id, glob){
                       user_id = glob$user$user_id)
         
         new_attribute_id <- dplyr::tbl(glob$pool, "attributes") %>% 
-          dplyr::filter(.data$attribute_name == local(input$attribute_name)) %>% 
-          dplyr::filter(.data$attribute_object == "user") %>% 
+          dplyr::filter(.data$project_id == !!as.numeric(glob$active_project), 
+                        .data$attribute_name == local(input$attribute_name), 
+                        .data$attribute_object == "user") %>% 
           dplyr::collect() %>% 
           dplyr::pull(attribute_id)
```

Comment:
i see what happend! would it not be safer to just do
```
attribute_ids <- dplyr::tbl(glob$pool, "attributes") %>% 
          dplyr::pull(attribute_id)
new_attribute_id <- max(attribute_ids)
```
?

Also, when using `pull()`, you do not need to use `collect()`.

Reply:
I think it's better to explicitly filter by attribute_name than depend on assuming that the attribute will have the max. ID (it could break in case that two users press the button at the same time maybe?). 
Fair point with `pull()`

Reply:
I am running into problems elsewhere with `log_add_document_record()` function that does not filter properly (probably an issue with `project_id`) and returns two or more integers if there are documents with the same name in the database (a likely scenario that could easily happen with attributes too). Finishing the pipeline of with `max()` solves the problem. I suggest to at least use it in that manner for more robustness.

Reply:
should be fixed now